lib/ansibledocgen/formatter/markup.py

```diff
@@ -27,7 +28,26 @@ def parse_data(self):
         """ Write Each Individual Markup File """
         for sourcefile in self.parserdata:
             self.write_doc(sourcefile)
-
+            
+    def __create_task_info_structure__(self, sourcefile):
+        '''
+        @author: Y_mil        
+        @contact: lylinquiman@gmail.com
+        @param sourcefile: sourcefile type
+        @return: All task name and tags in order for create markup
+        @rtype: list
+        This Function go through all taks and create the list for create markup        
+        '''
+        list_data = []        
+        for task_info in sourcefile['task_info']:
+            print(task_info['task_name'])
```

Comment:
Is this print statement needed or was it left over from debugging something?

Reply:
This print statement was used for debugging and see the avance. This can modify and use for check avance status. Whats you propourse?

Reply:
I think we can leave the print statement.